src/room.py

```diff
@@ -1,2 +1,49 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+class Room:
+    """
+    The Room object contains details about a room
+
+    Args: 
+        name (str): The rooms name
+        description (str): The rooms description
+
+    Attributes: 
+        n_to (Room): The room to the north
+        s_to (Room): The room to the south
+        e_to (Room): The room to the east
+        w_to (Room): The room to the west
+    """
+
+    def __init__(self, name, description):
+        self.name = name
+        self.description = description
+        self.n_to = None
+        self.s_to = None
+        self.w_to = None
+        self.e_to = None
+
+    def get_directions(self):
```

Comment:
Way to go Mark, to set up a dictionary as a compass and making your code more readable